frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -30,6 +30,9 @@ export const getAvailableClusterUpdates = (cv: ClusterVersionKind): ClusterUpdat
 export const getAvailableClusterChannels = () => ({
   'stable-4.3': 'stable-4.3',
   'fast-4.3': 'fast-4.3',
```

Comment:
We should be remove the 4.3 channels as well.

Reply:
Thanks, @spadgett.  Done.